site_files/site.NCAR

```diff
@@ -21,6 +21,7 @@ if (UNIX)
 	set (PYTHONVERSION 3.6)
 	set (PYTHONDIR ${THIRD_PARTY_DIR})
 	set (PYTHONPATH ${THIRD_PARTY_LIB_DIR}/python${PYTHONVERSION})
+	set (EXTRA_LIBS_SEARCH ${EXTRA_LIBS_SEARCH} dbus)
```

Comment:
Not really sure how cmake works, but Is this going to present a problem if dbus does NOT exist in EXTRA_LIBS_SEARCH, but would still be found in the system libraries?

Reply:
You say: "If possible, it is generally better to remove the dependency rather than include it this way."

Is there a preferable way to do this?

Reply:
dbus is an optional dependency of Qt so we could compile Qt without it, which would remove it as a dependency. It may not be worth recompiling Qt so we can postpone it by packaging dbus with the installer for now.

Reply:
Understood. Thanks.